parts/agentparams.t

```diff
@@ -52,6 +52,13 @@
       },
       "type": "string"
     },
+    "{{.Name}}WindowsImageSourceUrl": {
+      "defaultValue": "",
```

Comment:
reference the assigned value.  Looks at examples from Subnet or VMSize

Reply:
done.